src/sentry/search/snuba/backend.py

```diff
@@ -306,12 +312,15 @@ def do_search(project_id, environment_id, tags, start, end,
         else:
             conditions.append((col, '=', val))
 
-    aggregations = [
-        ['count()', '', 'times_seen'],
-        ['min', 'timestamp', 'first_seen'],
-        ['max', 'timestamp', 'last_seen'],
-        [priority_expr, '', 'priority']
-    ]
+    required_aggregations = set([sort] + extra_aggregations)
+    for h in having:
+        alias = h[0]
+        if alias in aggregation_defs:
```

Comment:
I didn't see any cases where one of these aliases *wouldn't* be in `aggregation_defs`. Did I misread this?

Reply:
Oops, you're right. When I started this I was using `conditions` before I remembered we now (have to) use having, which is all aggregates.